src/Symfony/Bundle/FrameworkBundle/Command/CacheWarmupCommand.php

```diff
@@ -74,6 +75,10 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $this->cacheWarmer->enableOptionalWarmers();
         }
 
+        if ($kernel instanceof WarmableInterface) {
+            $kernel->warmUp($kernel->getContainer()->getParameter('kernel.cache_dir'));
```

Comment:
It seems you can create the `$cacheDir` variable before this new `if` instead of line 82.